### ISSUE TITLE
WebContainer should be run locally in a docker container for More Privacy and accessibility to code

### ISSUE THREAD

**Issue by HomeDev68** [coleam00/bolt.new-any-llm#66]
**Is your feature request related to a problem? Please describe:**

Yes, the WebContainer loads in an online stackblitz container, which should be run locally for more Privacy, accessibility and to encourage offline use with local models

**Describe the solution you'd like:**

A local implementation of the WebContainer in which code is edited locally and previews and terminal commands are run in docker containers running locally.

**Describe alternatives you've considered:**

Probably the way the OpenCanvas Project by langraph loads files locally (although it's single file editing)

**Additional context:**

The addition of this feature would actually make this project better.
@coleam00 Could you add support for this, It would be nice. Thanks
